### ISSUE TITLE
layoutlmv3-base-chinese f1

### ISSUE THREAD

**Issue by rogerdehe** [microsoft/unilm#887]
**Describe the bug**
Model I am using (layoutlmv3-base-chinese):

The problem arises when using:
* [ ] the official example scripts: (give details below) https://github.com/microsoft/unilm/tree/master/layoutlmv3
* [ ] my own modified scripts: (give details below) https://colab.research.google.com/drive/1C-J668Cevdrn6KHo_NJ5urMzs1LRDDDH?usp=sharing

I cannot reproduce the `precision` `recall` `f1` in `zh XFUND`, do I miss something?

**To Reproduce**
1. run my colab code

**Expected behavior**
the expected result should close to 

Pre-trained Model | precision | recall | f1
-- | -- | -- | --
layoutlmv3-base-chinese | 0.8980 | 0.9435 | 0.9202

but I get 

Pre-trained Model | precision | recall | f1
-- | -- | -- | --
layoutlmv3-base-chinese | 0.8805 | 0.9266 | 0.9030

I find that you finetune with 8 GPUs, but I just have 1 GPU, so I have also tried to use `gradient_accumulation_steps=8` to simulation, I still cannot get expected result

Pre-trained Model | precision | recall | f1
-- | -- | -- | --
layoutlmv3-base-chinese | 0.8824 | 0.9436 | 0.9119

- Platform: linux
- Python version: 3.7
- PyTorch version (GPU?): 1.10.0+cu111

**Comment by Dod-o** [comment]
Hi @rogerdehe, because the xfund does not have the dev set, we choose the best score from the test set. I find your log output F1=91.6 during evaluation. So maybe the gap of 0.4 is because of the num of GPUs. 

`{'eval_loss': 1.1868419647216797, 'eval_precision': 0.8999052731291443, 'eval_recall': 0.9335080248935473, 'eval_f1': 0.9163987138263666, 'eval_accuracy': 0.8525394206087276, 'eval_runtime': 24.7395, 'eval_samples_per_second': 2.627, 'eval_steps_per_second': 0.364, 'epoch': 76.34}`

I think it is a good idea to use more GPUs or find another setting to get a similar score.
